source/ui/REIXS/REIXSSampleChamberButtonPanel.h

```diff
@@ -50,6 +52,7 @@ protected slots:
 
 	QToolButton *createQToolButton(QString text, QIcon icon);
 	AMControlMoveButton *createAMControlMoveButton(QString text, QIcon icon);
+	AMXYThetaControlMoveButton *createAMXYThetaControlMoveButton(QString text, QIcon icon);
```

Comment:
Just for my knowledg: this is just an optimization to prevent the compiler from replicating the strings and icons every time this method is called?